app/Models/School.ts

```diff
@@ -0,0 +1,42 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column, HasMany, hasMany } from '@ioc:Adonis/Lucid/Orm'
+import { jsonColumn } from 'App/Helpers/ModelJson'
+import Member from './Member'
+
+export type SchoolContact = {
+  email: string
+  phone: string
+}
+
+export type SchoolAddress = {
+  street: string
+  city: string
+  state: string
+  zip: string
+}
+
+export default class School extends BaseModel {
+  @column({ isPrimary: true })
+  public id: number
+
+  @column()
+  public name: string
+
+  @jsonColumn()
+  public contact: SchoolContact
+
+  @jsonColumn()
+  public address: SchoolAddress
```

Comment:
same - it would be better to have another model for the adress with a relation to the school (`hasOne` + `belongsTo`)